.format_text_tasks/.form

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Формы</title>
+</head>
+<body>
+    <table border="1" cellspacing="1" cellpadding="28">
+        <tr>
+            <td colspan="3">
+                <p>Ваше имя:</p>
+                <form>
+                    <input type="text">
+                </form>
+                <p>Ваше фото:</p>
+                <form>
+                    <input type="file">
+                </form>
+                <p>Введите пароль:</p>
+                <form>
+                    <input type="password">
+                </form>
+                <em>Это поле для тех, у кого особый пароль:</em>
+                <form>
+                    <input type="password">
+                </form>
+            </td>
+        </tr>
+        <tr>
+            <td colspan="3">
+                <h1>Пройдите тест</h1>
+            </td>
+        </tr>
+        <tr>
+            <td>
+                <h2>Любимое время года</h2>
+                <select>
+                    <option>зима</option>
+                    <option>весна</option>
+                    <option>лето</option>
+                    <option>осень</option>
+                    <option>все люблю</option>
+                    <option>все не люблю</option>
+                    <option>без разницы</option>
+                </select>
+            </td>
+            <td>
+                <h2>По какому режиму живете?</h2>
+                <p><input name="f" type="radio" >я 'сова'</p>
+                <p><input name="f" type="radio">я 'жаворонок'</p>
+                <p><input name="f" type="radio">без разницы</p>
```

Comment:
every form input should have unique name. Because then you will send the data in request to server using this names and if all names will be equal this lead to the errors